### DISCUSSION TITLE
No feedback from GitHub support, 10 days and counting

### DISCUSSION THREAD

**Discussion by franke-biosaxs** [community/community#49308] [New to GitHub]
Hi everyone.

10 days ago I created an account and an organisation, with the intend to subscribe to Teams. However, prior to doing so, I opened a ticket with support with questions on how billing works and whether there ware any options to import existing repositories and bugzilla entries for subscribers. So far, no feedback whatsoever. For the moment we are stuck and are uncertain how to proceed. Was support the wrong place to ask? I could not post here at the time as my account was blocked for being newly created?!

Any help or pointers would be appreciated. Thank you.

With kind regards

Daniel

**Comment by pratikkabade** [comment]
I too have bad experience with support, I guess they get too many requests so they might not be replying.

Either way about **billing** you can check all billings right from `your organisation`/ `setting`/`billing information`

And yes you can **import** existing repositories too all you have to do is go to your **repository settings** _scroll down_ and click on `transfer`

**Reply by franke-biosaxs** [reply]
Hi @pratikkabade. Thanks for your reply. Sorry, I was a bit vague on the billing. So far, we are not subscribed, so no billing. I am aware that it is 4USD per person per month. I wanted to know what counts as "person" in this regard. mostly: active developers only, or every member of the team? Right now our team would be at most three people, one active developer and at least one management-type "lurker", who would be there to post issues, follow progress etc. I do expect that GitHub would bill 2x4USD, but I wanted that confirmed.

With respect to import: tricky. The import wizard seems to require an URL. So far it is a self-hosted subversion repository that is not accessible via URL. Having that said, thank you for the pointer. The importer page linked some documentation that I hadn't seen/found yet. Shall read some more.

**Comment by queenofcorgis** [comment]
Hi there  @franke-biosaxs 👋🏼 

Welcome to the world of GitHub! Would you mind sharing your ticket number - I'm unable to escalate, but I can confirm it is in the right place. Support reply time is based on the amount of tickets ahead of you and our team's resources so we appreciate your patience. 

Cheers!

**Reply by queenofcorgis** [reply]
And re: not being able to post: We instituted a temporary interaction limit in order to reduce spammy behavior and keep our community healthy. Find the announcement [here](https://github.com/orgs/community/discussions/48334). 

If you have any feedback on this - we'd love to hear it 😄

**Reply by franke-biosaxs** [reply]
@queenofcorgis Thanks for your reply. The ticket number is #2028848. 

On the ban for new registrations: I can see why this might be necessary. Nonetheless, it took me an embarrassingly long time to figure that out. I spent at least half an hour trying to post something but gave up once I found a related footnote tucked at the bottom of a page somewhere. Phrased differently: just the absence of  the "New Discussion" button was insufficient to convey the message. A big fat red "you are banned from posting for the next 24 hours" note would have helped in that regard.

**Reply by queenofcorgis** [reply]
Hi there - I checked in on your ticket and it is in the right place in the queue for our Technical Support team, again the patience is appreciated, I know you've been waiting for quite some time. 

Thanks so much for your candid feedback on the interaction limits - I've passed this along to our team and we are already conversing about possible solutions!

**Comment by franke-biosaxs** [comment]
Just wanted to update my own question: eventually support did reply to my request and, oh boy, what a detailed answer! Most of my questions got resolved, I will still need to follow up with some details, though. Hooray!
